src/Maestro/MaestroApplication/ApplicationPackageRoot/ApplicationManifest.xml

```diff
@@ -132,6 +132,6 @@
     <DefaultRunAsPolicy UserRef="MaestroUser" />
   </Policies>
   <Certificates>
-    <EndpointCertificate X509FindType="FindBySubjectName" X509FindValue="[WebCertName]" Name="WebSslCert" />
+    <EndpointCertificate X509FindValue="[WebCertName]" Name="WebSslCert" />
```

Comment:
I didn't mean to change this... :/

Reply:
Yeah. VS auto-does this in this project for some reason.

Reply:
Stupid VS.